src/main/webapp/dev/js/common/bootboxWrapper.es6

```diff
@@ -15,25 +15,31 @@ const DEFAULT_CANCEL_TEXT = 'Cancel';
 const DEFAULT_YES_TEXT = 'Yes';
 const DEFAULT_NO_TEXT = 'No';
 
+function applyStyleForModal(modal, statusType) {
+    modal.find('.modal-header')
+        .addClass(`alert-${statusType || StatusType.DEFAULT}`)
```

Comment:
`statusType` shouldn't be optional here, though it requires some additional ugly code to make it non-optional.

Anyway, I prefer it to look something like this so it is clearer.

```es6
    modal.find('.modal-header').addClass(`alert-${statusType || StatusType.DEFAULT}`)
         .find('.modal-title').addClass(`icon-${statusType || StatusType.DEFAULT}`);
```